pkgs/development/python-modules/pre-commit/default.nix

```diff
@@ -0,0 +1,49 @@
+{ lib, buildPythonPackage, fetchPypi, pythonOlder
```

Comment:
This is already packaged as an application https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/version-management/git-and-tools/pre-commit/default.nix
Though this has some nice improvements.

Why was this added as a python-module?

Reply:
All is not lost, I will *borrow* some of these changes and incorporate them into nixpkgs somehow :smile: